x-pack/legacy/plugins/index_lifecycle_management/__jest__/components/edit_policy.test.js

```diff
@@ -238,22 +237,14 @@ describe('edit policy', () => {
     });
   });
   describe('warm phase', () => {
-    test('should show number required error when trying to save empty warm phase', () => {
```

Comment:
I think this is still a valid test. Is there a reason why you deleted it?

Reply:
I deleted it because we set a default value of 0 from the request below. Therefore, I don't think the test is valid anymore as will not have an empty value. 

> I am so stoked to see this get fixed! Thanks @jkelastic.
> 
> Can we improve the UX by setting the default value for "min age" to 0? This requires changing three files: `store/defaults/cold_phase.js` (line 20), `store/defaults/warm_phase.js` (line 26), and `store/defaults/cold_phase.js` (line 18). All three lines need to be changed to:
> 
> ```js
> [PHASE_ROLLOVER_MINIMUM_AGE]: 0,
> ```
> 
> Now when you go into the form, all of these inputs should be prepopulated with 0 and you should be able to save the form.

Reply:
I believe the user could still delete the default value, in which case, the validation would still occur.

Reply:
thank you, updated my code